File: PHP/AndandoNoTempoTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("AndandoNoTempo.php");

class AndandoNoTempoTest extends TestCase {

    public function testAndandoNoTempo() {
        $obj = new AndandoNoTempo();
        //Test 1
        $want = "S";
        $got = $obj->possibleTravel(22, 5, 22);
        $this->assertEquals($want, $got);
        //Test 2
        $want = "S";
        $got = $obj->possibleTravel(31, 110, 79);
        $this->assertEquals($want, $got);
        //Test 3
        $want = "N";
        $got = $obj->possibleTravel(45, 8, 7);
        $this->assertEquals($want, $got);
    }

}

File: JAVA/AndandoNoTempoTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class AndandoNoTempoTest {

	@Test
	public void testAndandoNoTempo() throws Exception {
		//Test 1
		String want = "S";
		String got = AndandoNoTempo.possibleTravel(22, 5, 22);
		assertEquals(want, got);
		//Test 2
		want = "S";
		got = AndandoNoTempo.possibleTravel(31, 110, 79);
		assertEquals(want, got);
		//Test 3
		want = "N";
		got = AndandoNoTempo.possibleTravel(45, 8, 7);
		assertEquals(want, got);
	}

}
